src/konnektor/network_planners/concatenator/_abstract_network_concatenator.py

```diff
@@ -19,8 +20,9 @@ def __init__(self, mapper: AtomMapper, scorer,
                  network_generator: _AbstractNetworkGenerator,
                  nprocesses: int = 1,
                  _initial_edge_lister=None):
-        """This class is an implementation for the LigandNetworkPlanner interface.
-        It defines the std. class for a Konnektor LigandNetworkPlanner
+        """This class is an implementation for the LigandNetworkPlanner
```

Comment:
ideally this first docstring line is a single line description of what this is.  The start of this line, "This class is an implementation for the" doesn't actually contain any content.  Better is possibly "Base class for Concatenator classes"

Reply:
jaaaa... that is a bad habit